transformations/vocab_restrictor/test.json

```diff
@@ -0,0 +1 @@
+{"type": "vocab_restrictor", "test_cases": [{"class": "VocabRestrictor", "inputs": {"sentence": "It takes great courage to see the world in all its tainted glory, and still to love it. -Oscar Wilde"}, "outputs": {"sentence": ["It takes great coragio to see the world in all its nettled preciousness, and still to love it. -Oscar Wilde", "It takes great braveness to see the world in all it untainteded glory, and still to love it. -Oscar Wilde", "It takes great braveness to see the world in all its fomented lord, and still to love it. -Oscar Wilde", "It takes great courage to see the world in all its tainted glory, and still to love it. -Oscar Wilde", "It takes great courage to see the world in all its tainted glory, and still to love it. -Oscar Wilde", "It takes great courage to see the world in all its scrawled glory, and still to love it. -Oscar Wilde", "It takes great wody to see the world in all its anointed sainte, and still to beloved it. -Oscar Wilde", "It takes great courage to see the world in all its cheered preciousness, and still to love it. -Oscar Wilde", "It takes great courage to see the world in all its anointed glory, and still to love it. -Oscar Wilde", "It takes great courage to see the world in all its unaskeded glory, and still to love it. -Oscar Wilde"]}}, {"class": "VocabRestrictor", "inputs": {"sentence": "If life were predictable it would cease to be life, and be without flavor. -Eleanor Roosevelt"}, "outputs": {"sentence": ["If life were opportune it would cease to be life, and be without relish. -Ariel Roosevelt", "If life were preposterously it would cease to be life, and be without fragrance. -Lily Roosevelt", "If life were forcible it would cease to be life, and be without bitter. -Owenite Roosevelt", "If life were forcible it would cease to be life, and be without flavour. -\u0152chalian Roosevelt", "If life were propitious it would cease to be life, and be without flavor. -Elne Roosevelt", "If life were expectantly it would stop to be life, and be without flavor. -Lawrence Roosevelt", "If life were uncertain it would cease to be life, and be without flavor. -Elengi Roosevelt", "If life were hopeful it would stop to be life, and be without flavour. -Elsie Roosevelt", "If life were improbable it would cease to be life, and be without taste. -Llyn Roosevelt", "If life were unexpectedly it would cease to be life, and be without taste. -Eliza Roosevelt"]}}]}
```

Comment:
Also you might want to reformat this json so that is clear to everyone reading it.